resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -2,7 +2,7 @@
 jruby-puppet: {
     # Where the puppet-agent dependency places puppet, facter, etc...
     # Puppet server expects to load Puppet from this location
-    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
+    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby, /opt/puppetlabs/puppet/cache/lib]
```

Comment:
This concerns me, because it means the environment that the agent (on the master) pluginsyncs & applies the catalog from, will affect how the master compiles catalogs for all other agents potentially in different environments.